packages/components/src/notice/test/index.tsx

```diff
@@ -120,4 +122,164 @@ describe( 'Notice', () => {
 			expect( speak ).toHaveBeenCalledTimes( 1 );
 		} );
 	} );
+
+	describe( 'actions', () => {
+		it( 'should render a disabled action button', () => {
```

Comment:
These tests might be a bit excessive in testing for all the different supported properties. Really all we care about is that props are passed through, right? So that's 1 test? At least for the ones where we're just looking at what happens with the button. It makes sense for the `should call onClick` regression test to be its own thing.

Same could said for the stories.

Reply:
I guess my concern is that this creates additional maintenance burden and potential for drift†, increases the runtime of tests, and doesn't really create much value over communicating that the actions can use any of the props from button and setting the expectation that they're passed through.

† On the topic of drift, there's already drift here because we only cover 5-6 of the available 15+ Button props.